lighthouse-core/report/report-generator.js

```diff
@@ -261,14 +262,16 @@ class ReportGenerator {
     });
 
     const template = Handlebars.compile(this.getReportTemplate());
+    reportContext = reportContext || 'extension'; // could be: devtools, extension
```

Comment:
it appears this is just used for styling, and just for devtools embed right now. I wonder if we should rename it to something to indicate that. `reportStyle` or whatever